homework-1/cash_register.py

```diff
@@ -0,0 +1,85 @@
+"""
+TASK 1
+
+Write a class to represent a Cash Register.
+You class should keep the state of price total and purchased items
+
+Below is a starter code:
+------------------------
+1. you can add new variables and function if you want to
+2. you will NEED to add accepted method parameters where required.
+For example, method add_item probably accepts some kind of an item?..
+3. You will need to write some examples of how your code works.
+"""
+
+
+class CashRegister:
+
+    def __init__(self):
+        self.total_items = None  # {'item': 'price'}
+        self.total_price = 0
+        self.discount = 0
+
+    def add_item(self, item, price):
+        if self.total_items is None:
+            self.total_items = {}
+        self.total_items[item] = price
+        self.total_price += price
+
+    def remove_item(self, item):
+        print('--- Removing {}--- '.format(item))
+        removed = self.total_items.pop(item)
+        self.total_price -= removed

```

Comment:
Hi Keiko; an exceptional logic you have implemented in this codebase. 
I wanted to point out an obvious implication of what you have done in the highlighted lines. 
In a real-world case, you may wanna provide logic that accounts for any error that may be encountered from retrieving data from your data structure. An if statement to check if the object exists could do, before directly trying to remove the user-provided object.

This has no direct mark reduction implication, I only wanted to point it out.